build.sbt

```diff
@@ -22,23 +22,27 @@ import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
 
 val appName = "digital-comms-dynamic-stub"
 
+scalaVersion := "2.12.8"
```

Comment:
This is what we did originally, but setting Scala to this version causes an Akka NoSuchMethodError. Changing the Play version doesn't fix it. Should I pin the Akka version to 2.5.23, the solution that the migration document (https://docs.google.com/document/d/1g8S5SiwuOQWDcd-2lHQfiF8qAxh2F9BgFC4yae_8TmQ/edit#) suggests?

Reply:
Did you try using Scala 2.11.12 and setting the Play plugin (in `plugins.sbt`) to 2.6.23? If you go higher than that (like 2.6.25 as its set to now) then that's when I've seen people get the issue.

Reply:
Yeah, I tried that and I still get the error

Reply:
Alright, there must be some other dependency conflict buried in there that we haven't encountered yet. To avoid digging through all of that I'm fine with this being on Scala 2.12.

I can see the other comments are marked as resolved so if you push the changes up I'll re-review the PR.